examples/mvc/README.md

```diff
@@ -0,0 +1,201 @@
+```python
```

Comment:
@ianmalcolm thanks for this. I think we can keep this README and remove the jupyter notebook.